.github/workflows/ci.yml

```diff
@@ -139,32 +138,3 @@ jobs:
       - name: Build
         run: |
           yarn react-native run-windows --release --arch x64 --logging --no-packager --no-launch --no-deploy --msbuildprops "BundleEntryFile=index.ts" --no-telemetry
-  release:
-    name: Release
-    needs: [review, android, ios, macos, windows]
-    runs-on: ubuntu-22.04
```

Comment:
Wouldn't it be simpler to add `if: ${{ github.event_name == 'push' }}` here?

Reply:
aw yeah, it would, but the idea of separate workflows looked more appealing to me 😄 
Let me change it back